core/themes/basis/css/component/fieldset.css

```diff
@@ -100,21 +103,27 @@ fieldset .fieldset-wrapper {
 }
 
 .fieldset-legend span.summary {
+  color: #757575; /* WCAG AA compliance */
```

Comment:
remove comment

Reply:
Removed with [`58a030d` (#2550)](https://github.com/backdrop/backdrop/pull/2550/commits/58a030d3d20c84360a342697a683b48e4bdd65d8). I'm adding a comment in the PR instead.